styles/core/utilities/typography/_font-declarations.scss

```diff
@@ -26,86 +26,86 @@
 // -------------------------
 
 @include addepar-font-face('Univers-Next',
-  '#{$univers-font-folder}/UniversNextW01Light', ('woff', 'ttf'), $font-light);
+  '#{$univers-font-folder}/UniversNextW01Light', ('woff2', 'woff'), $font-light);
```

Comment:
There's some additional cleanup we can do, now that this file only refs woff2 and woff fonts.

The `addepar-font-face` mixin has special-casing for eot, ttf and otf fonts that we could drop. (Actually, we could drop the "otf" now because we don't already don't have any otf fonts in this repo):

https://github.com/Addepar/addepar-style-toolbox/blob/80f216c477387fc181c3e7e6e54f63ed208cf6c9/styles/core/utilities/typography/_mixins.scss#L50-L57

And we could/should also drop the eot, ttf and otf font files from this repo, too, as part of this PR or in a follow-up.

Reply:
I'm going to leave the code for now. We still use this library in some odd places. I'm not convinced those spots will ever upgrade to the newer 1.x series I'm cutting here, but regardless I don't think we need to make that potential path and more difficult than it already might be.

For example in concierge https://github.com/Addepar/it-tools/blob/056412d263d236afce3d63097887cc67d6d06218/concierge/frontend/app/styles/base/_typography.scss#L38 and in navigator https://github.com/Addepar/navigator-core/blob/9792ae944a60b6d9a558f0047a3ee771daa5b591/static/interface/stylesheets/customizing/_addepar_fonts_overwrite.scss#L4 (although I think they might have a fork of the mixin)